pkg/crypto/tinkcrypto/primitive/composite/ecdhes/ecdhes_aes_aead_private_key_manager.go

```diff
@@ -21,42 +21,42 @@ import (
 )
 
 const (
-	ecdhesPrivateKeyVersion = 0
-	ecdhesPrivateKeyTypeURL = "type.hyperledger.org/hyperledger.aries.crypto.tink.EcdhesAeadPrivateKey"
+	ecdhesAESPrivateKeyVersion = 0
+	ecdhesAESPrivateKeyTypeURL = "type.hyperledger.org/hyperledger.aries.crypto.tink.EcdhesAesAeadPrivateKey"
 )
 
 // common errors
-var errInvalidECDHESPrivateKey = fmt.Errorf("ecdhes_private_key_manager: invalid key")
-var errInvalidECDHESPrivateKeyFormat = fmt.Errorf("ecdhes_private_key_manager: invalid key format")
+var errInvalidECDHESAESPrivateKey = fmt.Errorf("ecdhes_aes_private_key_manager: invalid key")
+var errInvalidECDHESAESPrivateKeyFormat = fmt.Errorf("ecdhes_aes_private_key_manager: invalid key format")
 
-// ecdhesPrivateKeyManager is an implementation of PrivateKeyManager interface.
-// It generates new ECDHESPrivateKey keys and produces new instances of ECDHESAEADCompositeDecrypt subtle.
-type ecdhesPrivateKeyManager struct{}
+// ecdhesAESPrivateKeyManager is an implementation of PrivateKeyManager interface.
+// It generates new ECDHESPrivateKey (NIST AES EC) keys and produces new instances of ECDHESAEADCompositeDecrypt subtle.
```

Comment:
drop "NIST" - use the actual curve names. Saying NIST is ambiguous.

Reply:
done